blocks/search-results/search-results.js

```diff
@@ -0,0 +1,42 @@
+import { createCardBlock } from '../card/card.js';
+import { loadBlock } from '../../scripts/lib-franklin.js';
+
+export async function searchResults(params, jsonData) {
+  // eslint-disable-next-line
+    return jsonData.data.filter((entry) => (entry.title + entry.content + entry.path + entry.authors + entry.collections + entry.section + entry.categories + entry.template).toLowerCase().includes(params.toLowerCase()));
```

Comment:
instead of disabling lint, make it easier to read. also `template` is not needed. 
```suggestion
  return jsonData.data.filter((entry) => (
    entry.title
    + entry.content
    + entry.path
    + entry.authors
    + entry.collections
    + entry.section
    + entry.categories
  )
    .toLowerCase()
    .includes(params.toLowerCase()));
```

Reply:
done